client/src/components/Lyrics.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const LyricsSection = styled.section`
```

Comment:
put all styles in a seperated file. if its 1 - it can be in the same file, if it more - move it